src/pages/edge/edge-network/tutorials/send-datastream-overrides.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Sending datastream overrides to Edge Network
+description: Learn how to 
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Sending datastream overrides to Edge Network
+
+Datastream overrides let you define additional configuration for your datastreams, allowing you to trigger different datastream behaviors without either creating a new datastream or modifying your existing settings.
```

Comment:
I don't entirely understand this sentence since for config ID override you need to have a new datastream created. Maybe something like this:

Datastream overrides let you define additional configuration settings for your datastreams, allowing you to trigger different datastream behaviors on a per event basis.

Reply:
I honestly don't know what's going on with datastreams, so I copied the original sentence from the datastreams guide 😂 

If you can provide better info for this, that would be greatly appreciated.

Reply:
you can use the text highlighted above.